tests/ui/debuginfo/dwarf-versions.rs

```diff
@@ -0,0 +1,32 @@
+//@ revisions: one two three four five six
+
+//@[one] compile-flags: -Zdwarf-version=1
```

Comment:
Span-less errors can be checked with `error-pattern`, we should do that here.

```suggestion
//@[one] compile-flags: -Zdwarf-version=1
//@[one] error-pattern: requested DWARF version 1 is not supported
```